example-application/test/basic-tests.test.js

```diff
@@ -72,29 +91,33 @@ describe("/api", () => {
       });
 
       //Act
-      const receivedAPIResponse = await request(expressApp).post("/order").send(orderToAdd);
+      const addedOrderId = (await request(expressApp).post("/order").send(orderToAdd)).body.id;
+      const retreivedAddedOrder = await request(expressApp).get("/order/" + addedOrderId);
```

Comment:
@DanielGluskin This is part of the assert phase. Act phases contains always 1 call against the method/route under test